resource.cpp

```diff
@@ -493,6 +493,8 @@ static const ProductMap products[] =
     {"_TZ3000_ltiqubue", "TS130F", "Tuya", "Zigbee curtain switch"},
     {"_TZ3000_vd43bbfq", "TS130F", "Tuya", "QS-Zigbee-C01 Module"}, // Curtain module QS-Zigbee-C01
     
+    {"_TZ3000_m0vaazab", "TS0207", "Tuya", "Zigbee Repeater"},
```

Comment:
I'd like to prefix the product id as the other devices, something like:

```
{"_TZ3000_m0vaazab", "TS0207", "Tuya", "Tuya_RPT Repeater"},
```
In general I'm not so fancy of having Zigbee in the product name, because... they are all Zigbee devices :D


That way `R_GetProductId(&lightNode) == QLatin1String("Tuya_RPT Repeater")` might be more clear that this is a Tuya device?